.editorconfig

```diff
@@ -0,0 +1,79 @@
+[*]
```

Comment:
I assume this file is PyCharm config.
Could you add comment in the beginning of the file, explaining what it is?

Reply:
Actually no, it's a general config format. That said it has some special settings for pycharm too.

Reply:
Done